tools/cmake/README.md

```diff
@@ -82,27 +82,56 @@ If you're running CMake directly, you may need to pass it in yourself as follows
 cmake -S <source-dir> -B <build-dir> -DCMAKE_BUILD_TYPE=debug
 ```
 
-## How to build a greentea test
-
-Install prerequisites suggested in the previous section and follow the below steps to build:
-* Set your current directory to the test suite directory
-
-* Run the following command for the configuration CMake module to be generated
-  ```
-  mbedtools configure -t <TOOLCHAIN> -m <MBED_TARGET> --mbed-os-path /path/to/mbed-os
-  ```
-* Build the test binary with the full profile
+## How to build and run greentea tests
```

Comment:
Perhaps add some information about the CMake variable to pass on the CLI to set reset delay `MBED_GREENTEA_TEST_RESET_TIMEOUT`

Reply:
Something like:

If forcing a reset using option `-r` in `MBED_HTRUN_ARGUMENTS`, you can set up after reset idle delay by passing the CMake variable `-`DMBED_GREENTEA_TEST_RESET_TIMEOUT`=<time_in_seconds>`.

Reply:
Seems like a layering violation to set it that way. We want that variable to be set by mbed_config.cmake, it only exists so the Mbed config settings are respected. If there's no config system to set it, the reset timeout flag should just be added to `MBED_HTRUN_ARGUMENTS` like any other argument (using `-R/--reset-timeout=NUMBER`).

Reply:
Ah! I did not know it was generated by mbed-tools. It is fine then.